controllers/process.js

```diff
@@ -2,6 +2,9 @@
 
 const config = require(__dirname + '/../config')
 const Staticman = require('../lib/Staticman')
+let staticman;
```

Comment:
The codebase is following [StandardJS](http://standardjs.com/), which doesn't use semi-colons. Could you change that, please?